AsyncDisplayKit/Private/ASStackUnpositionedLayout.mm

```diff
@@ -23,15 +23,16 @@
                                   const CGFloat stackMin,
                                   const CGFloat stackMax,
                                   const CGFloat crossMin,
-                                  const CGFloat crossMax)
+                                  const CGFloat crossMax,
+                                  const CGSize size)
 {
   const ASStackLayoutAlignItems alignItems = alignment(child.alignSelf, style.alignItems);
   // stretched children will have a cross dimension of at least crossMin
   const CGFloat childCrossMin = alignItems == ASStackLayoutAlignItemsStretch ? crossMin : 0;
   const ASSizeRange childSizeRange = directionSizeRange(style.direction, stackMin, stackMax, childCrossMin, crossMax);
-  ASLayout *layout = [child measureWithSizeRange:childSizeRange];
+  ASLayout *layout = [child layoutThatFits:childSizeRange parentSize:size];
```

Comment:
If we have proper nullability annotations for `-layoutThatFits:parentSize`, it will be clear that `layout` can't be nil and so we will neither need to assert nor handle the nil case.